app/styles/app/layouts/profile.scss

```diff
@@ -668,6 +668,43 @@ header.space-between {
       }
     }
 
+    &.migratable {
+      opacity: 1.0;
+
+      .migration-status {
+        width: 66.5156px;
```

Comment:
I have never considered fractional values for pixels, but it seems that it might actually change something: https://stackoverflow.com/a/9080685/661200

That's a minor thing (as it will most likely only be visible with high zoom levels), but it's an interesting thing to know.